charts/kueue/templates/crd/kueue.x-k8s.io_clusterqueues.yaml

```diff
@@ -177,31 +177,36 @@ spec:
                     before borrowing or preempting in the flavor being evaluated.
                   properties:
                     whenCanBorrow:
-                      default: Borrow
+                      default: ReadyToUse
                       description: |-
                         whenCanBorrow determines whether a workload should try the next flavor
                         before borrowing in current flavor. The possible values are:
 
-                        - `Borrow` (default): allocate in current flavor if borrowing
+                        - `ReadyToUse` (default): allocate in the current flavor if borrowing
```

Comment:
Nit: If this is correct https://github.com/kubernetes-sigs/kueue/issues/6187#issuecomment-3363305862

Lets update the docs.

`ReadyToUse` (default): allocate in the current flavor if borrowing is possible.
-> 
`ReadyToUse` (default): allocate in the current flavor if borrowing allows workload to fit in the current flavor.